routers/filter_ssr.go

```diff
@@ -62,22 +73,60 @@ func InitChromeDp() {
 	}
 }
 
+func cacheSave(md5urlString string, res string) {
+	renderCache[md5urlString] = SSRCache{time.Now(), res}
+}
+
+func cacheRestore(md5urlString string) (string, bool) {
+	if _, ok := renderCache[md5urlString]; ok {
+		if time.Now().Sub(renderCache[md5urlString].time) < time.Duration(cacheExpirationTime)*time.Second {
+			return renderCache[md5urlString].html, true
+		}
+	}
+	return "", false
+}
+
+func url2md5(urlString string) string {
+	md5url := md5.New()
+	md5url.Write([]byte(urlString))
+	md5urlString := fmt.Sprintf("%x", md5url.Sum(nil))
+	return md5urlString
+}
+
+func initCacheSettings() {
+	isCacheSettingsInit = true
+	var err error
+	cacheExpirationTime, err = beego.AppConfig.Int64("cacheExpirationTime")
```

Comment:
You can call `beego.AppConfig.Int64("cacheExpirationTime")` in any time at run-time, no need to init